app/views/general/_frontpage_intro_sentence.html.erb

```diff
@@ -1,6 +1,3 @@
-<h1 class="intro__title">
-  <%= _("Get answers from the government and public sector") %>
-</h1>
-<p class="intro__description">
-  <%= _("Make a request for information to a public authority: by law, they have to respond") %>
-</p>
+<h1 class="intro__title"><%= _("Your right to know") %></h1>
+<p class="intro__description"><%= _("Every citizen has the right to access information held by public authorities. By law, they have to respond") %></p>
```

Comment:
Since you're working here, we could do:

```erb
<p class="intro__description">
  <%= _("Every citizen has the right to access information held by public authorities.") %>
  <% if AlaveteliConfiguration.authority_must_respond %>
    <%= _("By law, they have to respond.") %>
  <% end %>
</p>

```